### ISSUE TITLE
userdata-bbb_skip_check_audio_on_first_join is not working after BBB_getEchoTest is set to true

### ISSUE THREAD

**Issue by galupa** [bigbluebutton/bigbluebutton#14132]
**Describe the bug**
Hi,

I am not sure if this is a bug or not. I tested this on BBB 2.4.1.
I tried using userdata-bbb_skip_check_audio_on_first_join parameter for create a room and it looks great, it skipping the echo test. However after I manually trigger the echo test (by turn off audio and turn it on again), echo test is not skipping again after I end the meeting and create a new one.

**To Reproduce**
1. Create a room using `userdata-bbb_skip_check_audio_on_first_join=true`. It should be something like `https://bbb/bigbluebutton/api/join?createTime=1642539050734&fullName=name&join_via_html5=true&meetingID=some_id&password=passwd&userID=id&userdata-bbb_skip_check_audio_on_first_join=true&checksum=some_checksum`
2. Echo test will be skipped
3. Click leave audio (I noticed `BBB_getEchoTest` in session storage is changing from false to true)
4. End meeting
5. Rejoin / restart the meeting
6. Echo test is shown

What I noticed is echo test will be skipped again if `BBB_getEchoTest` is set to false manually or if I totaly clean up the session on my browser.

**Expected behavior**
I expect on step #6 the echo test is not shown

**Actual behavior**
On step #6 echo test is shown.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser Firefox
 - Version 96.0 

**Additional context**

**Comment by ramonlsouza** [comment]
Hello @galupa, I could not reproduce the error by following the steps you provided. 
When I end the meeting (step 4) and start again, echo test is skipped as expected... 

I also tried to just copy the url (https://bbb/html5client/join?sessionToken=xxxx) and open in another tab, and echo test was skipped again (session storage was not kept).

The only way to "leave" the meeting and keep `BBB_getEchoTest` value is by refreshing the page.

Could you please give more information about steps 4-5? How are you rejoining / restarting the meeting after ending it?

**Comment by galupa** [comment]
Hi @ramonlsouza,

Thank you for the reply, I just tried to reproduce again on chromium browser version 97.0.4692.71. What I noticed is `userdata-bbb_skip_check_audio_on_first_join=true` needs to be part of the url request when join / create the meeting.

I recorded what I have done that might be of help, I also will try to reproduce it on other machine later if I can get hold on other machine soon.

![Screencast from 20-01-22 10 02 19](https://user-images.githubusercontent.com/29434066/150214318-c3af8640-4b79-49bd-ae8e-5fe41a3fc6c3.gif)

Cheers,
